test/integration/targets/pip/tasks/url_packages.yml

```
@@ -0,0 +1,199 @@
+- name: Create local package
+  vars:
+    repo_path: "{{ remote_tmp_dir }}/pip_vcs_pkgs"
+  block:
+  - name: Create a nested package directory layout
+    file:
+      path: "{{ repo_path }}/dummy-root/dummy-subdirectory"
+      recurse: yes
+      state: directory
+      mode: '0755'
+
+  - name: Create a root-level pyproject.toml
+    copy:
+      dest: "{{ repo_path }}/pyproject.toml"
+      content: |
+        [build-system]
+        requires = ["setuptools>=61.0"]
```

It's usually better readable if the spec is sparse.
```suggestion
        requires = ["setuptools >= 61.0"]
```